RockPaperScissors.rb

```diff
@@ -0,0 +1,79 @@
+=begin
+Abigail and Benson are playing Rock, Paper, Scissors.
+
+Each game is represented by an array of length 2, where the first element represents what Abigail played and the second element represents what Benson played.
+
+Given a sequence of games, determine who wins the most number of matches. If they tie, output "Tie".
+
+R stands for Rock
+P stands for Paper
+S stands for Scissors
+
+Examples
+calculate_score([["R", "P"], ["R", "S"], ["S", "P"]]) ➞ "Abigail"
+
+# Ben wins the first game (Paper beats Rock).
+# Abigail wins the second game (Rock beats Scissors).
+# Abigail wins the third game (Scissors beats Paper). 
+# Abigail wins 2/3.
+
+calculate_score([["R", "R"], ["S", "S"]]) ➞ "Tie"
+
+calculate_score([["S", "R"], ["R", "S"], ["R", "R"]]) ➞ "Tie"
+
+=end
+
+def who_wins(player1, player2)
+
+  if player1 > player2
+    "Abigail" 
+  elsif player1 == player2 
+    "Tie"
+  else
+    "Ben"
+  end
+end
+
+def calculate_score(game)
+  abi = 0
+  ben = 0
+
+  if game.length > 0
```

Comment:
you can make this more modular. with a new method that figures out which wins. rick, paper or scissors

Reply:
this code is too complex. need to simplify into smaller methods

Reply:
Complex in the sense ?